code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -270,11 +270,6 @@
 
 	return 1
 
-/mob/living/carbon/get_standard_pixel_y_offset()
```

Comment:
Macro this may be?

Reply:
How would I macro that since it takes both `lying` and the `initial` of pixel_y?